tag/client.go

```diff
@@ -0,0 +1,88 @@
+package tag
+
+import (
+	"context"
+
+	"github.com/mongodb/grip"
+	"github.com/mongodb/grip/message"
+	"github.com/pkg/errors"
+
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/resourcegroupstaggingapi"
```

Comment:
I usually don't rename packages unless they cause a package name clash though, because it makes it difficult to keep a consistent package name (unless you always remember to rename it). But yeah, this is a common issue with AWS's code-generated SDK.